ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/underlay_routing_protocol.schema.yml

```diff
@@ -27,4 +28,4 @@ keys:
       - "isis-ldp"
       - "isis-sr-ldp"
       - "ospf-ldp"
-      - "BGP" # Only supported for historic reasons - The code will change to the default
+      - "bgp" # Only supported for historic reasons - The code will change to the default
```

Comment:
do we still need `bgp`? if so could the commend make it to the doc?

Reply:
there is another pr incoming from Carl, which will remove the need for this. So everything is up in the air these days :) Please just let this one in first.

Reply:
Correction, let's get #2691 in first, and then we rebase this one.